src/storage/txn/task.rs

```diff
@@ -17,6 +17,7 @@ use crate::storage::{
     },
 };
 
+#[derive(Debug)]
```

Comment:
Does this debug would print out the key value of `Prewrite`?

Reply:
Yes if its inner command is not taken out, like

[scheduler.rs:764] ["process cmd with snapshot"] [**task**="Task { cid: 10, tracker_token: TrackerToken { shard_id: 43, seq: 3, key: 0 }, cmd: Some(kv::command::pessimistic_prewrite mutations([(Put **key**:7480000000000000FF865F728000000000FF00000C0000000000FA **value**:8000010000000202004901 assertion:NotExist, DoPessimisticCheck)]) primary(7480000000000000865F72800000000000000C) secondary_len(Some(0))@ 451561100694257665 3002 1 451561100694257667 451561101219332097 true Fast (for_update_ts constraints: []) | region_id: 3003 region_epoch { conf_ver: 1 version: 69 } peer { id: 3004 store_id: 1 } term: 42 max_execution_duration_ms: 20000 resource_group_tag: 0A209505CACB7C710ED17125FCC6CB3669E8DDCA6C8CD8AF6A31F6B3CD64604C30981801 request_source: \"external_Commit\" resource_control_context { resource_group_name: \"default\" penalty {} override_priority: 8 } keyspace_id: 4294967295 source_stmt { connection_id: 2428502020 } cluster_id: 7349211160867591748), extra_op: Noop, owned_quota: Some(OwnedAllocated { allocated: 475, from: MemoryQuota { in_use: 8571, capacity: 268435456 } }) }"] [extra_op=Noop] [term=Some(42)] [cid=10] [thread_id=70]

Reply:
Mostly we don't care about value, can we skip it, otherwise the log could be too large

Reply:
To troubleshoot certain correctness issues such as the row value codec, we may need to precisely know the row value carried by the request and the row value written into the kv db.

Printing all the KV pairs would indeed make the logs too large. Currently, we usually choose to customize logging selectively when reproducing issues to address specific problems.

Reply:
In this case, I wonder whether it should be in the debug level. It would result in the case that the log may be even larger than the data. I'm afraid we may need a higher level of logging, or some special logging configuration for this.

Reply:
@MyonKeminta 
How about skipping the values in this PR, and find a proper way to display these costy details latter?

Reply:
> How about skipping the values in this PR, and find a proper way to display these costy details latter?

I'm okay with this idea.

Reply:
```
[scheduler.rs:764] ["process cmd with snapshot"] [task="Task { cid: 3, tracker_token: TrackerToken { shard_id: 52, seq: 1, key: 0 }, cmd: Some(kv::command::pessimistic_prewrite mutations([(Put key:7480000000000000FF865F728000000000FF0001570000000000FA value:skipped
````

Apply `cmd_batch` length is printted.

["raft log is applied to the kv db"] **[cmd_batch=1]** [req_info="Some(RequestInfo { region_id: 23001, start_ts: 451816127840124929, task_id: 0, resource_group_tag: [10, 32, 149, 5, 202, 203, 124, 113, 14, 209, 113, 37, 252, 198, 203, 54, 105, 232, 221, 202, 108, 140, 216, 175, 106, 49, 246, 179, 205, 100, 96, 76, 48, 152, 24, 1], request_type: KvPrewrite, cid: 3, is_external_req: true })"] [thread_id=207]